nuclear/tools/module.py

```diff
@@ -107,15 +133,20 @@ class {className} : public NUClear::Reactor {{
         {closeNamespace}
 
         #endif  // {define}
-        """)
+        """
+    )
+
+    return template.format(
+        define='{}_H'.format('_'.join([p.upper() for p in parts])),
+        className=parts[-1],
```

Comment:
A wild camelCase appeared

Reply:
Pretty sure these are your fault, I just formatted the code lol